### PR TITLE
Install xdebug also on 7.2

### PR DESCRIPTION
**Pull request by shyim**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -49,6 +49,7 @@ RUN apk add --no-cache \
       apcu \
       imagick \
       redis && \
```
Comment (jeboehm) [inline Dockerfile]: Please remove the && :)

**Review by jeboehm — CHANGES_REQUESTED** [review]
Hi @shyim 
thank you for your contribution. Could you fix the small bug in your commit?

File: Dockerfile (hunk 1 above)
Comment (shyim) [reply Dockerfile]: ups 😄

**Comment by jeboehm** [comment]
<3

**Pull request merged**